librclone/php/rclone.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/*
+PHP interface to librclone.so, using FFI ( Foreign Function Interface )
+
+Create an rclone object
+
+$rc = new Rclone( __DIR__ . '/librclone.so' );
+
+Then call rpc calls on it
+
+    $rc->rcp( "config/listremotes", "{}" );
+
+When finished, close it
+
+    $rc->close();
+*/
+
+class Rclone {
+
+    protected $rclone;
+
+    function __construct( $libshared )
+    {
+        $this->rclone = FFI::cdef("
+        struct RcloneRPCResult {
+            char* Output;
+            int	Status;
+        };        
+        extern void RcloneInitialize();
+        extern void RcloneFinalize();
+        extern struct RcloneRPCResult RcloneRPC(char* method, char* input);
+        extern void RcloneFreeString(char* str);
+        ", $libshared);
+        $this->rclone->RcloneInitialize();
+    }
+
+    function rcp($method, $input ): array
```

Comment:
Can you rename this to `rpc` for remote procedure call?

Reply:
yes, of course, aaahhhh! rcp!

Now in our internal tests with phpunit, have discovered a small bug, that it will be fixed also. Give us a little time.
Later, in real life,  if we discover something wrong, also we will contribute in your amazing project. 

Thanks, for your work in this project.